stdlib/public/core/AnyHashable.swift

```diff
@@ -262,6 +262,10 @@ extension AnyHashable: CustomReflectable {
 
 @available(macOS 9999, iOS 9999, tvOS 9999, watchOS 9999, *)
 extension AnyHashable: _HasCustomAnyHashableRepresentation {
+}
+
+extension AnyHashable {
```

Comment:
It might be worthwhile to add a FIXME comment here in case we can clean it up in the future.